src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +119,214 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
 					?.SendFlyoutItemsChanged();
 		}
 
-		/// <summary>Bindable property for attached property <c>TabBarIsVisible</c>.</summary>
+		/// <summary>
+		/// Manages the tabbar visibility.
+		/// The tab bar and tabs are visible in Shell applications by default. 
+		/// </summary>
 		public static readonly BindableProperty TabBarIsVisibleProperty =
 			BindableProperty.CreateAttached("TabBarIsVisible", typeof(bool), typeof(Shell), true);
 
-		/// <summary>Bindable property for attached property <c>TitleView</c>.</summary>
+		/// <summary>
+		/// Enables any View to be displayed in the navigation bar.
+		/// </summary>
 		public static readonly BindableProperty TitleViewProperty =
 			BindableProperty.CreateAttached("TitleView", typeof(View), typeof(Shell), null, propertyChanged: OnTitleViewChanged);
 
-		/// <summary>Bindable property for attached property <c>MenuItemTemplate</c>.</summary>
+		/// <summary>
+		/// Customize the appearance of each MenuItem.
+		/// </summary>
 		public static readonly BindableProperty MenuItemTemplateProperty =
 			BindableProperty.CreateAttached(nameof(MenuItemTemplate), typeof(DataTemplate), typeof(Shell), null, BindingMode.OneTime);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetMenuItemTemplate']/Docs/*" />
+		/// <summary>
+		/// Gets the DataTemplate applied to MenuItem objects in the MenuItems collection.
```

Comment:
Cref classes you reference, also see my comment about parameters: https://github.com/dotnet/dotnet-api-docs/wiki/Parameters